Cargo.toml

```diff
@@ -16,8 +16,22 @@ resolver = "2"
 
 [features]
 default = ["ws", "rustls"]
+
+# Network protocols
 http = ["dep:reqwest", "dep:tokio-util", "dep:indexmap"]
 ws = ["dep:tokio-tungstenite", "dep:tokio-stream", "dep:indexmap", "tokio/time"]
+
+# Embedded database backends
+mem = ["surrealdb/kv-mem"]
```

Comment:
it maybe work with `mem = ["surrealdb/kv-mem","tokio/io-util"]`
due to some error `unresolved import `tokio::io::AsyncReadExt` no `AsyncReadExt` in `io``

Reply:
Thanks for the review. I had already fixed this locally.